clients/q.js

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env node
+
+const crypto = require('crypto');
+const http = require('http');
+
+const trimQuery = sql => sql.trim().replace(/(?:\r\n|\r|\n)/g, " ");
+
+const hash = sql => crypto.createHash('sha256').update(sql).digest('hex');
+
+const hostname = 'localhost';
+const port = 8080
```

Comment:
We should be able to override these, e.g. to query `https://demo.seafowl.io` or a custom deployment.